R/app_startup.R

```diff
@@ -42,12 +50,19 @@ app_startup<-function(domainData=NULL, meta=NULL, charts=NULL, mapping=NULL, cha
         mapping<-bind_rows(mapping_list, .id = "domain")
     }
 
+    # Set filterDomain to null if no data exists
+    if(!filterDomain %in% names(domainData)){
```

Comment:
Only issue with this is if `filterDomain == NULL` this check bombs.  Obviously that's never going to actually happen outside of testing but worth noting.

Reply:
Hm, I think this is actually worth fixing. I don't think it would be crazy for someone to call `safetyGraphicsApp(filterDomain=NULL)` to initialize the app with no filtering. And that does indeed bomb on this line. 

Good catch!